docs/tutorials/multimodal/object_detection/quick_start/quick_start_coco.md

```diff
@@ -0,0 +1,204 @@
+
+```python
+zip_file = "s3://automl-mm-bench/object_detection_dataset/tiny_motorbike_coco.zip"
+download_dir = "./tiny_motorbike_coco"
+
+load_zip.unzip(zip_file, unzip_dir=download_dir)
+data_dir = os.path.join(download_dir, "tiny_motorbike")
+train_path = os.path.join(data_dir, "Annotations", "coco_trainval.json")
+test_path = os.path.join(data_dir, "Annotations", "coco_test.json")
+```
+
+While using COCO format dataset, the input is the json annotation file of the dataset split.
+In this example, `coco_trainval.json` is the annotation file of the train-and-validate split,
+and `coco_test.json` is the annotation file of the test split.
+
+We select the YOLOv3 with MobileNetV2 as backbone,
+and input resolution is 320x320, pretrained on COCO dataset. With this setting, it is fast to finetune or inference,
+and easy to deploy.
+For more model choices, see :label:`sec_automm_detection_selecting_models`.
+And we use all the GPUs (if any):
+
+```python
+checkpoint_name = "yolov3_mobilenetv2_320_300e_coco"
+num_gpus = -1  # use all GPUs
+```
+
+We create the MultiModalPredictor with selected checkpoint name and number of GPUs.
+We need to specify the problem_type to `"object_detection"`,
+and also provide a `sample_data_path` for the predictor to infer the catgories of the dataset.
+Here we provide the `train_path`, and it also works using any other split of this dataset.
+And we also provide a `path` to save the predictor. 
+It will be saved to a automatically generated directory with timestamp under `AutogluonModels` if `path` is not specified.
+
+```python
+# Init predictor
+predictor = MultiModalPredictor(
+    hyperparameters={
+        "model.mmdet_image.checkpoint_name": checkpoint_name,
+        "env.num_gpus": num_gpus,
+    },
+    problem_type="object_detection",
+    sample_data_path=train_path,
+    path="./quick_start_tutorial_temp_save",
+)
+```
+
+We set the learning rate to be `2e-4`.
+Note that we use a two-stage learning rate option during finetuning by default,
+and the model head will have 100x learning rate.
+Using a two-stage learning rate with high learning rate only on head layers makes
+the model converge faster during finetuning. It usually gives better performance as well,
+especially on small datasets with hundreds or thousands of images.
+We also set the epoch to be 15 and batch_size to be 32.
+For more information about how to tune those hyperparameters,
+see :ref:`sec_automm_detection_tune_hyperparameters`.
+We also compute the time of the fit process here for better understanding the speed.
+
+```python
+start = time.time()
+# Fit
+predictor.fit(
+    train_path,
+    hyperparameters={
+        "optimization.learning_rate": 2e-4, # we use two stage and detection head has 100x lr
+        "optimization.max_epochs": 15,
+        "env.per_gpu_batch_size": 32,  # decrease it when model is large
+    },
+)
+train_end = time.time()
+```
+
+We run it on a g5dn.12xlarge EC2 machine on AWS,
```

Comment:
Do we want to attach console logs here or let it run on CI? I feel this one is a good (maybe only) example that should be run on CI.

Reply:
Sure I'll do it.